### PR TITLE
Fix for #26612

### LINKED ISSUE #26612: Casting not necessary
Tell me if I'm wrong, but in `tensorflow/lite/kernels/internal/reference/integer_ops/softmax.h`, is casting to <int 32> necessary? (Line 89 & 90)

### PR DESCRIPTION
**Pull request by kyscg**
Fixed #26612

### CODE REVIEW

**Comment by kyscg** [comment]
Is this good @rthadur ?????

**Comment by kyscg** [comment]
I'm closing this, casting is unnecessary
